### PR TITLE
#349: update stories to typescript

### PR DESCRIPTION
**Pull request by mob8607**
## Pull request
Refactors storybook stories to typescript.
 
### Ticket
#349 
 
### Browser testing
- http://localhost:6006/?path=/docs/elements-e-button--docs
 
### Checklist
- [x] I merged the current development branch (before testing)
- [ ] Added JSDoc and styleguide demo
- [x] Tested all links in project relevant browsers
- [x] Tested all links in different screen sizes
- [x] Did run automated tests and linters
- [ ] Did assign ticket
- [x] Double checked target branch
 
## Review/Test checklist
- [ ] Did review code and documentation
- [ ] Tested all links in project relevant browsers
- [ ] Tested all links in different screen sizes
- [ ] Did check accessibility (Wave, only errors)
- [ ] Re-assign ticket to developer

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,11 +7,13 @@
   "scripts": {
     "clean:caches": "rm -rf .eslintcache .stylelintcache node_modules/.cache",
     "dev": "vite",
-    "lint": "npm run lint:eslint && npm run lint:stylelint",
+    "fix:eslint": "npm run lint:eslint -- --config .eslintrc.fix.cjs --cache=false --fix",
+    "fix:stylelint": "npm run lint:stylelint -- --cache=false --config .stylelintrc.fix.js --fix",
```
Comment (benjamin-joham) [inline package.json:11]: question: are these triggered in the pre-commit hook or just here for manual execution?

File: src/stories/elements/e-checkbox.stories.ts [hunk 2]
```diff
@@ -12,8 +13,9 @@ export default {
 };
 
 // More on component templates: https://storybook.js.org/docs/vue/writing-stories/introduction#using-args
-const Template = {
-  render: (args) => ({
+const Template: StoryObj = {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  render: (args: any) => ({
```
Comment (benjamin-joham) [inline src/stories/elements/e-checkbox.stories.ts:18]: note: I don't like having any here. But I think for now it is sufficient enough.

**Review by benjamin-joham — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (mob8607) [reply package.json:11]: They are for manual execution

File: src/stories/elements/e-checkbox.stories.ts (hunk 2 above)
Comment (mob8607) [reply src/stories/elements/e-checkbox.stories.ts:18]: I couldn't get it to work without it. Maybe I missed something here.

**Pull request merged**